frappe/utils/fixtures.py

```diff
@@ -47,8 +47,12 @@ def export_fixtures():
 	"""Export fixtures as JSON to `[app]/fixtures`"""
 	for app in frappe.get_installed_apps():
 		for fixture in frappe.get_hooks("fixtures", app_name=app):
-			print "Exporting {0}".format(fixture)
+			filters = None
+			if isinstance(fixture, dict):
+				filters = fixture.get("filters")
+				fixture = fixture.get("doctype") or fixture.get("dt")
+			print "Exporting {0} app {1} filters {2}".format(fixture, app, filters)
```

Comment:
I don't see the error. fixture is defined, app also and filters is defined and will print None when don't enter inside if. I'am wrong?